lib/constants.js

```diff
@@ -1,16 +1,23 @@
+/* eslint-disable no-template-curly-in-string */
 const path = require('path');
 module.exports = {
   experience: {
     commandType: 'experience',
     apiType: 'experienceViews',
     localStatusParams: [ path.join(path.sep, '{components,layouts,pages}', '*.hbs') ],
-    remoteStatusParams: [ path.join('experience', '${viewType}s', '${name}.hbs'), 'body' ] // eslint-disable-line no-template-curly-in-string
+    remoteStatusParams: [ path.join('experience', '${viewType}s', '${name}.hbs'), 'body' ]
   },
   files: {
     commandType: 'files',
     apiType: 'files',
     localStatusParams: [ path.join(path.sep, '**') ],
-    remoteStatusParams: [ 'files${parentDirectory}${name}', 's3etag', { skipMd5Creation: true } ] // eslint-disable-line no-template-curly-in-string
+    remoteStatusParams: [ 'files${parentDirectory}${name}', 's3etag', { skipMd5Creation: true } ]
+  },
+  dataTables: {
+    commandType: 'dataTables',
+    apiType: 'dataTables',
+    localStatusParams: [ path.join(path.sep, '*.csv' )],
+    remoteStatusParams: [ path.join('dataTables', '${name}.csv') ]
```

Comment:
names of data tables are not unique, I would append the id just so we don't overwrite the same file.